1.7/administration/high-availability/index.md

```diff
@@ -0,0 +1,29 @@
+---
+post_title: High-Availibility 
+menu_order: 3.1
+---
+
+DC/OS supports high-availability configurations, including mult-zone, multi-region, and hybrid cloud. 
+
+# Terminology
+
+## Rack
```

Comment:
Rack here seems a bit out of place. Because it is not clear what is its relation between zones and regions. I think part of the confusion is that we trying to mix on-prem and public-cloud terms.

I'm thinking maybe it's less confusing if we have separate pages for on-prem, public-cloud and hybrid configurations?